qcut/resources/default-skills/organize-project/REFERENCE.md

```
@@ -0,0 +1,225 @@
+│   ├── Skill.md
+│   ├── REFERENCE.md
+│   ├── CONCEPTS.md
+│   └── ADVANCED.md
+└── custom-skill/         # User-created skills
+    └── Skill.md
+```
+
+## File Type Mappings
+
+| Extension | Category | Destination |
+|-----------|----------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)